compiler/kernel/expr.go

```diff
@@ -403,9 +403,21 @@ func compileCall(zctx *resolver.Context, scope *Scope, call ast.Call) (expr.Eval
 	case call.Name == "exists":
 		exprs, err := compileExprs(zctx, scope, call.Args)
 		if err != nil {
-			return nil, fmt.Errorf("exists: bad argument: %w", err)
+			return nil, fmt.Errorf("exists(): bad argument: %w", err)
```

Comment:
Looks like error strings here are inconsistent when it comes to adding this `name():` prefix.

Maybe name the return parameters and add then add the prefix in a `defer` ?

```go
defer func() {
    if err != nil {
        err = fmt.Errorf("%s(): %w", call.Name, err)
    }
}()    
```

Reply:
Maybe I'm doing something wrong, but this involves a bit of mess getting the scope right everywhere on err, then defining locally scoped vars for each non-error value.  I think I'll just make it consistent for now and we can rework it later.  Also, I find code with named returned values harder to read compared to explicit return values.

Reply:
Does the trouble stem from using unadorned returns? I'm no fan of those either, but you don't have to use them in order to have named result parameters.

Reply:
I will commit as is and you can show me the light...